interfaces/builtin/media_control.go

```diff
@@ -0,0 +1,59 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+// See https://www.kernel.org/doc/Documentation/userspace-api/media/mediactl/media-controller-intro.rst
+const mediaControlSummary = `allows access to media control devices`
+
+// The kernel media controller allows connecting and configuring
+// media hardware subsystems
```

Comment:
could you add the same comment about disallowing auto-connection since configuring/controlling the devices is considered a privileged operation?

Reply:
Comment added, thanks for the review!